### DISCUSSION TITLE
Chia wallet

### DISCUSSION THREAD

**Discussion by shourya080** [swar/Swar-Chia-Plot-Manager#1087] [Q&A]
Where do you add your wallet address so that you come to know whether you’ve received chia or not from your plots while using plot manager??

**Comment by w4d3j** [comment]
You don't.  Read the docs.

https://www.chia.net/2021/02/22/plotting-basics.html
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
https://github.com/Chia-Network/chia-blockchain/wiki/How-to-Check-If-Everything-is-Working-(or-Not)
